config/gulp/doc-util.js

```diff
@@ -55,11 +43,11 @@ module.exports = {
 
   logError(name, message) {
     log(colors.red(name), colors.yellow(message));
-    notify(`${this.dirName} gulp ${name}`, message, true);
+    notify.display(`${this.dirName} gulp ${name}`, message, true);
```

Comment:
Are these kept so we can fix later? Not seeing how `notify.display()` works w/ the function being deleted.